code/modules/projectiles/guns/misc/medbeam.dm

```diff
@@ -33,6 +33,8 @@
 	..()
 	LoseTarget()
 
+//this is an embarassing excuse for documentation, but this is just so people in the future don't fuck it all up with this shitcode
```

Comment:
this isn't the right format for a dmdoc comment, needs to be triple slashes.

Reply:
Bee is right about this, check the documentation guide for more details.